### ISSUE TITLE
very read-y

### ISSUE THREAD

**Issue by eadehemingway** [oliverjam/intro-react-workshop#3]
would be better if this was broken up by mini presentations instead of having us read so much for so long

**Comment by oliverjam** [comment]
Yeah I think you're right. There's not a super great way to just learn a whole library API in one go, at some point you just have to read/learn all the methods before you start actually writing code.

I might put points for everyone to stop so we can go through stuff together
